.github/workflows/conda-envs/oldest-test-reqs.txt

```diff
@@ -1,5 +1,5 @@
 ase==3.16.0
-cmake==3.12.0
```

Comment:
I wouldn't put `pip` requirements files in a directory named `conda-envs`. Developers would expect to find conda environment files in such a directory.

Reply:
I could rename the directory `env-files`, or something like that.